elasticsearch/elasticsearch.rkt

```diff
@@ -39,7 +39,11 @@
 (struct client (host port ssl? username password))
 
 ;;; Constants
-(define DEFAULT-CLIENT (client "localhost" 9200 #f #f #f))
+(define ES-USERNAME (if (getenv "ES_USERNAME") (getenv "ES_USERNAME") #f))
```

Comment:
You could just do `(define ES-USERNAME (getenv "ES_USERNAME"))` and for ES-PASSWORD.